.gitignore

```diff
@@ -28,5 +28,7 @@ __pycache__/
 htmlcov/
 _dynamic/
 coverage.xml
+coverage-*.xml
 diffcov.html
+diffcov-*.html
```

Comment:
The current `pytest` runner has been configured to provide testenv-specific coverage result files.